pyproject.toml

```diff
@@ -43,6 +43,131 @@ spark = ["googleapis-common-protos == 1.56.4", "grpcio >= 1.48, < 1.57", "grpcio
 sql = ["connectorx", "sqlalchemy", "sqlglot"]
 unity = ["httpx <= 0.27.2", "unitycatalog"]
 viz = []
+dev = [
+    # Development/Build utilities
```

Comment:
It's better to use [dependency-groups] other than extras to manage the dev dependencies